src/common/low_precision_transformations/tests/batch_to_space_transformation.cpp

```diff
@@ -0,0 +1,168 @@
+        ngraph::builder::subgraph::DequantizationOperations dequantization_before;
+        ov::element::Type preicsionAfterOperation;
+        ngraph::builder::subgraph::DequantizationOperations dequantization_after;
+    };
+
+    TestTransformationParams params;
+    std::vector<size_t> block_shape;
+    std::vector<size_t> crops_begin;
+    std::vector<size_t> crops_end;
+    Actual actual;
+    Expected expected;
+};
+
+typedef std::tuple<
+    ngraph::PartialShape,
+    BatchToSpaceTransformationTestValues> BatchToSpaceTransformationParams;
+
+class BatchToSpaceTransformation : public LayerTransformation,
+                                   public testing::WithParamInterface<BatchToSpaceTransformationParams> {
+public:
+    void SetUp() override {
+        const ngraph::PartialShape input_shape = std::get<0>(GetParam());
+        const BatchToSpaceTransformationTestValues test_values = std::get<1>(GetParam());
+
+        actualFunction = ngraph::builder::subgraph::BatchToSpaceFunction::get(
+            input_shape,
+            test_values.actual.input_type,
+            test_values.actual.dequantization_before,
+            test_values.block_shape,
+            test_values.crops_begin,
+            test_values.crops_end,
+            test_values.actual.dequantization_after);
+
+        SimpleLowPrecisionTransformer transform;
+        transform.add<ngraph::pass::low_precision::BatchToSpaceTransformation>(test_values.params);
+        transform.transform(actualFunction);
+
+        referenceFunction = ngraph::builder::subgraph::BatchToSpaceFunction::get(
+            input_shape,
+            test_values.expected.input_type,
+            test_values.expected.dequantization_before,
+            test_values.block_shape,
+            test_values.crops_begin,
+            test_values.crops_end,
+            test_values.expected.dequantization_after);
+    }
+
+    static std::string getTestCaseName(testing::TestParamInfo<BatchToSpaceTransformationParams> obj) {
+        const ngraph::PartialShape shape = std::get<0>(obj.param);
+        const BatchToSpaceTransformationTestValues testValues = std::get<1>(obj.param);
+
+        std::ostringstream result;
+        result << testValues.actual.input_type << "_"<<
+            shape << "_" << toString(testValues.params) << "_" <<
+            testValues.actual.dequantization_before << "_" <<
+            testValues.actual.dequantization_after << "_" <<
+            testValues.expected.dequantization_before << "_" <<
+            testValues.expected.dequantization_after << "_";
+        return result.str();
+    }
+};
+
+TEST_P(BatchToSpaceTransformation, CompareFunctions) {
+    actualFunction->validate_nodes_and_infer_types();
+
+    const auto testValues = std::get<1>(GetParam());
+    const auto output_precision = LayerTransformation::get_output_precision<ngraph::opset2::BatchToSpace>(actualFunction);
+    ASSERT_EQ(testValues.expected.preicsionAfterOperation, output_precision);
```

Comment:
I suppose we check the output precision in `compare_functions`, don't we? Why do we need a separate check then?